### PR TITLE
fix(docs): slightly enhancing ai lab documentation

### PR DESCRIPTION
**Pull request by myfear**
### What does this PR do?

Slightly improve readability of the ai lab extension docs.

### Screenshot / video of UI

n/a

### What issues does this PR fix or reference?

n/a

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/docs/ai-lab/installing.md [hunk 1]
```diff
@@ -8,10 +8,13 @@ tags: [ai, llm, generative ai]
 
 # Installing the Podman AI Lab extension
 
+The AI Lab extension extends the capabilities of Podman Desktop for large language models. It has to be installed before you can use it. Installations is just a click if you have Podman Desktop installed already.
```
Comment (jeffmaury) [inline website/docs/ai-lab/installing.md]:
```suggestion
The AI Lab extension extends the capabilities of Podman Desktop for large language models. It has to be installed before you can use it. Installation is just a click if you have Podman Desktop installed already.
```

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by axel7083 — APPROVED** [review]

File: website/docs/ai-lab/index.md [hunk 2]
```diff
@@ -8,10 +8,13 @@ tags: [ai, llm, generative ai]
 
 # Run LLMs locally
 
-Podman AI Lab can help you run large language models (LLMs) locally with ease.
+Podman AI Lab is the easiest way to work with Large Language Models (LLMs) on your local developer workstation. Find a catalog of recipes, leverage a curated list of open source models, experiment and compare the models. Get ahead of the curve and take your development to new heights wth Podman AI Lab!
+There are many ways to run models locally. This extension fits perfectly into your local container workflow and exposes LLMs through inference APIs that you can directly access from your application containers. Beyond that you can use playgrounds to optimze your inference parameters and recips that help you with ready made examples.
```
Comment (feloy) [inline website/docs/ai-lab/index.md]:
```suggestion
There are many ways to run models locally. This extension fits perfectly into your local container workflow and exposes LLMs through inference APIs that you can directly access from your application containers. Beyond that you can use playgrounds to optimze your inference parameters and recipes that help you with ready made examples.
```

**Review by feloy — APPROVED** [review]

File: website/docs/ai-lab/installing.md [hunk 3]
```diff
@@ -8,10 +8,13 @@ tags: [ai, llm, generative ai]
 
 # Installing the Podman AI Lab extension
 
+The AI Lab extension extends the capabilities of Podman Desktop for large language models. It has to be installed before you can use it. Installation is just a click if you have Podman Desktop installed already.
```
Comment (benoitf) [inline website/docs/ai-lab/installing.md]:
```suggestion
The AI Lab extension extends the capabilities of Podman Desktop for Large Language Models (LLMs). It has to be installed before you can use it. Installation is just a click if you have Podman Desktop installed already.
```

looks like you wrote with capital letters elsewhere

for the last part of the sentence, should it be `Installation is just a click away if you already have Podman Desktop installed.`

**Review by slemeur — APPROVED** [review]
Nice improvements!
LGTM

**Pull request merged**